about.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Oscafest</title>
+		<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <header>
+		<div class="header--social">
+			<ul>
+				<li><a href="#"><i class="bi bi-facebook"></i></a></li>
+        <li><a href="#"><i class="bi bi-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+			</ul>
+		</div>
+		<div class="header--nav--section">
+			<div class="header--hamburger">
+				<img src="images/hamburger.png" alt="hamburger-icon">
+			</div>
+			<a href="./index.html" class="header--logo">
+				<div class="logo--design">
+						<p>OS</p>
+				</div>
+				<p class="logo--text">OS Summit <br>2023</p>
+			</a>
+			<div class="header--nav--container">
+				<nav class="header--nav">
```

Comment:
- [ ] nice work. Please remember, we are obliged to follow the design for now. You can add modifications later on. Kindly remove the logo and social nav from the mobile version, so it can align with the original design. 
yours
![Screenshot (1190)_LI](https://github.com/Akorede3133/oscafest-conference/assets/65298048/5a9ad89b-1fce-491e-92d6-2def4539a3ed)
original
![Screenshot (1002)](https://github.com/Akorede3133/oscafest-conference/assets/65298048/16dee63c-ac30-4697-947c-3cd02f466b4c)